modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3065,6 +3065,67 @@ definition {
 		}
 	}
 
+	@description = "LPS-173537 - Verify on the Account object if it's possible to trigger an action with ADD when ADD an account"
+	@priority = 5
+	test CanTriggerAddActionWithSystemObjectToItselfWhenAdd {
+		property portal.acceptance = "true";
+
+		task ("Given a Account object and given an action to add a new Account entry after creating an Account entry") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 180195",
+				objectName = "CustomObject180195",
+				pluralLabelName = "Custom Objects 180195");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Integer",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "Integer",
+				isRequired = "false",
+				objectName = "CustomObject180195");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject180195");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 180195");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				actionName = "CustomAction",
+				active = "true",
+				inputMethod = "true",
+				inputValue = "Object entry added",
+				objectLabel = "Account",
+				thenAction = "Add an Object Entry",
+				whenAction = "On After Add",
+				inputSecondMethod = "true",git 
+				inputSecondValue = "customField");
```

Comment:
These steps on Custom Object are not needed.

`Given 1:` Go to Account Object
`Given 2:` Add action on Account
`When:` Create Account
`Then:` Assert new Account created